aggregations/db_tables/deployed_contracts.py

```diff
@@ -60,3 +81,102 @@ def start_of_range(self, timestamp: int) -> int:
     @staticmethod
     def prepare_data(parameters: list, *, start_of_range=None, **kwargs) -> list:
         return parameters
+
+    def store(self, parameters: list) -> list:
+        super().store(parameters)
+
+        near_rpc_url = os.getenv("NEAR_RPC_URL")
+        if not near_rpc_url:
+            return
+        near_rpc = near_api.providers.JsonProvider(near_rpc_url)
+
+        def view_code(account_id, block_id):
+            for _ in range(1000):
+                try:
+                    response = near_rpc.json_rpc(
+                        "query",
+                        {
+                            "request_type": "view_code",
+                            "account_id": account_id,
+                            "block_id": block_id,
+                        },
+                    )
+                except near_api.providers.JsonProviderError as e:
+                    if e.args[0].get("cause", {}).get("name") == "UNKNOWN_ACCOUNT":
+                        return b""
+                    print("Retrying fetching contract code...")
+                    traceback.print_exc()
+                except Exception:
+                    print("Retrying fetching contract code...")
+                    traceback.print_exc()
+                else:
+                    break
+            return base64.b64decode(response["code_base64"])
+
+        sql_missing_sdk_type = """
+            SELECT contract_code_sha256, deployed_to_account_id, deployed_at_block_hash
+            FROM deployed_contracts
+            WHERE contract_sdk_type = '' AND deployed_at_block_hash != ''
+            LIMIT 10
```

Comment:
Why do we need 10 such contracts? Let's leave a comment about that

Reply:
I finished reading, got the idea.
I suggest reloading `collect` instead of `store`. I see no benefit from inserting and then selecting and updating here.

Reply:
I will change it to 100, but there is no way to integrate it into collect as we collect all the deployments of a day and rely on the unique index in the database to deduplicate the contracts that were deployed earlier.

Reply:
Ah, I see